primitives/src/locktime/relative.rs

```diff
@@ -105,6 +105,23 @@ impl LockTime {
     #[inline]
     pub const fn from_height(n: u16) -> Self { LockTime::Blocks(Height::from_height(n)) }
 
+    /// Try to interpret the given number as a relative lock time.
+    #[inline]
+    pub fn from_num(num: i64) -> Option<LockTime> {
```

Comment:
In fd78f83e7e4f2fb6dcbd5ce8ca555b436973f72b:

How does this differ from `from_consensus` except that it does a conversion to `u32` for you?

Reply:
Removed. Good point!